geode-docs/managing/security/implementing_authentication.html.md.erb

```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
 
-        - Set property `security-client-auth-init` to the fully-qualified
-        class name that implements the `AuthInitialize` interface:
+  ``` pre
+  security-client-auth-init=com.example.security.ClientAuthInitialize.create
+  ```
 
-        ``` pre
-        security-client-auth-init=com.example.security.ClientAuthInitialize
-        ```
-        - Set property `security-client-auth-init` to the fully-qualified
-        name of a static method that instantiates an object of the class
-        that implements the `AuthInitialize` interface:
+Implement the `getCredentials` method of the `AuthInitialize` interface for the client to acquire values for
+the `security-username` and `security-password` properties in whatever way
+wish. For example, it might look up values in a database or another external resource,
+or it might prompt for values.
 
-        ``` pre
-        security-client-auth-init=com.example.security.ClientAuthInitialize.create
-        ```
 
-    - Implement the `getCredentials` method of the `AuthInitialize` interface
-    for the client.
-    The implementation of `getCredentials` acquires values for
-    the `security-username` and `security-password` properties in whatever way
-    it wishes.
-    It might look up values in a database or another external resource,
-    or it might prompt for values.
+### <a id="authentication-provideclientcredaccess"></a>Provide Secure Access to Client Credentials
 
-- Set the `security-username` and `security-password` properties for the client
-  in a way that can be accessed by the `getCredentials` implementation in the
-  `AuthInitialize`. This can be done via the APIs, properties file or other external
-  sources:
+Set the `security-username` and `security-password` properties for the client in a way that can be
+accessed by the `getCredentials` implementation in `AuthInitialize`. This can be done via the APIs,
+properties file or other external sources:
 
-    ``` pre
-    Properties properties = new Properties();
-    properties.setProperty("security-username", "exampleuser23");
-    properties.setProperty("security-password", "xyz1234");
-    ClientCache cache = new ClientCacheFactory(properties).create();
-    ```
+``` pre
+Properties properties = new Properties();
+properties.setProperty("security-username", "exampleuser23");
+properties.setProperty("security-password", "xyz1234");
+ClientCache cache = new ClientCacheFactory(properties).create();
+```
 
-    Take care that credentials set in this manner are not accessible
-    to observers of the code.
- 
+For security, take care that credentials set in this manner are not accessible to observers of the code.
 
 
-## How Other Components Set Their Credentials
+## <a id="authentication-component-set-creds"></a>How Other Components Set Their Credentials
 
-`gfsh` prompts for the user name and password upon invocation of
+`gfsh` prompts for the username and password upon invocation of
```

Comment:
add, "and these username/password combination will be provided as properties to the `authenticate` method in the keys of `security-username` and `security-password`".